whynot/simulators/hiv/environments.py

```diff
@@ -59,6 +60,8 @@ def observation_space():
     reward_fn=get_reward,
 )
 
-register(
-    id="HIV-v0", entry_point=HivEnv, max_episode_steps=400, reward_threshold=1e10,
+gymnasium.register(
+    id="HIV-v0", entry_point=HivEnv, apply_api_compatibility=True, max_episode_steps=400, reward_threshold=1e10,
 )
+
+env = gymnasium.make("HIV-v0", apply_api_compatibility=True)
```

Comment:
This is the line I have to add so gymansium understands the environment was defined.
As mentioned in the PR body I'm not sure how to register new environments with the gymnasium API. It's a next step.